lib/galaxy/jobs/runners/pulsar.py

```diff
@@ -378,6 +378,9 @@ def queue_job(self, job_wrapper):
                     remote_pulsar_app_config.update(yaml.safe_load(fh))
             job_directory_files = []
             config_files = job_wrapper.extra_filenames
+            if externalized_command:
+                config_files.append(externalized_command)
```

Comment:
I think you should have the same effect if you add `config_files.append(tool_script)` after line 387

Reply:
Ah yes, much simpler and it works :)